hypershift-operator/controllers/nodepool/nto.go

```diff
@@ -414,6 +417,9 @@ func BuildMirrorConfigs(ctx context.Context, cg *ConfigGenerator) ([]*MirrorConf
 func getMirrorConfigForManifest(manifest []byte) (*MirrorConfig, error) {
 	scheme := runtime.NewScheme()
 	_ = mcfgv1.Install(scheme)
+	_ = v1alpha1.Install(scheme)
+	_ = configv1.Install(scheme)
```

Comment:
It seems like only this line (configv1) is require to fix the bug.  Why add the hypershift and config v1alpha1 types as well?